### ISSUE TITLE
PyEZ crashes just before the 'load' but after connection is made

### ISSUE THREAD

**Issue by sgurumurthy** [Juniper/py-junos-eznc#113]
Pyez crashes with the following error if the connectivity with the device is lost while the my python script is in the middle of execution (The connection to the device is made but just before 'load' the connectivity is lost).

[root@esbu-mflab-lnx-c09 python2.7-installables]# python2.7 test.py 
Traceback (most recent call last):
  File "test.py", line 22, in <module>
    rsp = dev.cu.load(set_commands, format='set')
  File "/usr/local/lib/python2.7/site-packages/jnpr/junos/utils/config.py", line 217, in load
    return self.rpc.load_config( rpc_contents, **rpc_xattrs )
  File "/usr/local/lib/python2.7/site-packages/jnpr/junos/rpcmeta.py", line 74, in load_config
    return self._junos.execute( rpc )
  File "/usr/local/lib/python2.7/site-packages/jnpr/junos/device.py", line 280, in execute
    rsp = JXML.remove_namespaces( err.xml )
AttributeError: 'TimeoutExpiredError' object has no attribute 'xml'

Basically PyEZ needs to catch this error and raise an exception to the client calling it.
